sdk/template/aztemplate/version.go

```diff
@@ -0,0 +1,15 @@
+//go:build go1.17
+// +build go1.17
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcore
```

Comment:
CI builds are failing because you've got the wrong package name.

```suggestion
package aztemplate
```